cmd/pebble/cmd_help.go

```diff
@@ -168,11 +168,15 @@ type helpCategory struct {
 var helpCategories = []helpCategory{{
 	Label:       "Basics",
 	Description: "basic management",
-	Commands:    []string{"run", "autostart", "start", "stop", "exec", "services", "logs"},
+	Commands:    []string{"run", "exec"},
```

Comment:
A couple of things here:

* I added the new `signal` command for this commit.
* I'm adding in `restart` and `replan`, missed here when those were added.
* "basic management" is getting too long, especially with the "checks" command that will be added in another PR. I think the category "service management" is good, though I'm not sure about the "basic management" name anymore -- suggestions for a better name/grouping?

Reply:
Thanks for fixing my mistake with restart/replan

Reply:
Also I like this grouping